### DISCUSSION TITLE
Datatable Column: colspan in tbody?

### DISCUSSION THREAD

**Discussion by MiBaWheels** [primefaces/community#392] [PrimeVue] [answered]
Hi
I'm searching for a solution for the DataTable Component... The goal is, to have two header for one Column/field.

![test](https://github.com/primefaces/community/assets/125953511/ba36f666-36a6-4a3f-99cc-d2b1f5c3ca5f)

On the Top right of my thead Element, I want to display the total length of rows in my DataTable. In the screenshot I have added the colspan attribute in the tbody (only in first 2 rows) via browser Inspektor.

Is there a way to automatic apply it to the tbody? I only find solutions with colspan's in the thead element.

Sorry for my bad english and thank you for the beautiful vue-package!

Regards
Mike

**Comment by TeoDar** [comment, marked as answer]
Perhaps [this example](https://stackblitz.com/edit/zocvxj?file=src%2FApp.vue) will help you. I used Column slot [#header](https://primevue.org/datatable/#api.column.slots.header) to place additional data in the column header, while swapping the title block and line count data between the props header and #header slot. I also [used :pt](https://primevue.org/datatable/#pt.doc.column) to place slot data and props in the corners.

**Reply by MiBaWheels** [reply]
Hi TeoDar,
This solution would be fine, I also got this solution before:
 ```
        <Column field="lastlogin" sortable>
            <template #header>
                <div class="d-flex justify-content-between w-100 p-column-title">
                    <span>Last Login</span>
                    <span>{{ datatable ? datatable.processedData.length : 0 }}</span>
                </div>
            </template>
        </Column>
```

The Problem with this solution is, that the sorticon is then placed wrong (in my opinion).
What i get:
![grafik](https://github.com/primefaces/community/assets/125953511/1da25db5-3cba-4bf1-af13-88dbcf66e253)
What i want:
![grafik](https://github.com/primefaces/community/assets/125953511/13761349-1837-4708-80db-c97bca63d031)

But thanks for your solution

**Reply by TeoDar** [reply]
Try looking at my example again. I added "order" to headerTitle style. Get closer to your desired result. Perhaps some more CSS improvements will be needed...

**Reply by MiBaWheels** [reply]
That looks fine for me too... at the first look. So there is no need for a little component change.
Thank you for the answer and solution!
